hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1507,64 +1507,79 @@
       name: chip_otbn_op
       desc: '''Verify an OTBN operation.
 
-            SW test directs the BIGNUM engine to perform an operation. The BIGNUM SW image is
-            backdoor loaded into OTBN IMEM. SW validates the reception of the otbn done interrupt.
-            SW also verifies the correctness of the OTBN operation using a reference model. Details
-            TBD.
+            - SW test directs the OTBN engine to perform an ECDSA operation.
+            - SW validates the reception of the otbn done interrupt once the operation is complete.
+            - SW also verifies the correctness of the result using the reference model generated
+              result.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_otbn_imem_uncorrectable_alert
-      desc: '''Verify the imem uncorrectable alert from OTBN.
+      name: chip_otbn_rnd_entropy
+      desc: '''Verify OTBN can fetch RND numbers from the entropy src.
 
-            Inject 2 bit errors within the IMEM SRAM inside OTBN via backdoor and ensure that this
-            alert propagates to an NMI.
+            - SW initializes the entropy subsystem to generate randomness.
+            - SW loads an OTBN app that executes instructions to read the RND bits.
+            - The OTBN app ensures that the values when read consequtively do not match, and its not
+              all 0s or all 1s, as a basic measure to ensure that the entropy subsystem is returning
+              some data.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_otbn_dmem_uncorrectable_alert
-      desc: '''Verify the dmem uncorrectable alert from OTBN.
+      name: chip_otbn_urnd_entropy
+      desc: '''Verify OTBN can fetch URND numbers from the entropy src.
 
-            Inject 2 bit errors within the DMEM SRAM inside OTBN via backdoor and ensure that this
-            alert propagates to an NMI.
+            - Similar to chip_otbn_rnd_entropy, but verifies the URND bits.
```

Comment:
Seems like a good idea. I'm not sure how the mapping from "a test you run" to the hjson works, but I'd probably suggest squashing this test with the RND one: they'll share loads of common code and it seems a bit silly to duplicate the setup/teardown time.

Reply:
While each testpoint must cater to a unique design feature for the ease of testplan review (to judge completeness), we could very well have the same 'developed test' cater to multiple testpoints. The reverse is also true - a testpoint may need to be broken down into multiple developed tests. That's what the `tests` key is for - mapping the testpoints with developed tests.

A 'developed test' is what you can run with dvsim.